src/locales/en/common.json

```diff
@@ -232,6 +232,7 @@
   "Estimated reward": "Estimated reward",
```

Comment:
NRTTL
When editing the multisignatur account it skips the enter password step?

https://github.com/LiskHQ/lisk-desktop/assets/8784876/7683cc3e-b909-464b-ba63-c5a0f73c1708

Reply:
So for this I think its okay. Reason because it seems the current account who wants to edit the multi-signature isn't a signatory to the account any more and since all params signatures needs to signed before all root signatures can be collected, having the user sign would be useless because we wouldn't use his signature for anything until all params signatures have been collected @oskarleonard .

Reply:
Make sense